lomap/mcs.py

```diff
@@ -419,7 +419,7 @@ def delete_broken_ring():
                 moli_idx = int(at.GetProp('to_moli'))
                 moli_at = self._moli_noh.GetAtomWithIdx(moli_idx)
                 molj_idx = int(at.GetProp('to_molj'))
-                molj_at = self._moli_noh.GetAtomWithIdx(moli_idx)
+                molj_at = self._molj_noh.GetAtomWithIdx(molj_idx)
```

Comment:
this looks like it's fixing a bug which was in the code since it was written, I'll have to double check this though

Reply:
Since `ringMatchesRingOnly=True` and `completeRingsOnly=True` is set when calling `rdFMCS.FindMCS`, the `delete_broken_ring` has no chance to remove any atom.
Thus, AFAIK, these 2 typo fixes have no side effect but correcting the business logic:)

Reply:
Because atoms which fail a distance check are removed **after** the `FindMCS ` call, it is possible for partial rings despite these arguments being provided to FindMCS.

Reply:
Get it.
If `moli_at.IsInRing()` is True, The mapped one `molj_at.IsInRing()` must be True as well.
Thus, the behavior of `delete_broken_ring` keeps the same with my fix.